AppController/djinn.rb

```diff
@@ -1376,6 +1380,36 @@ def set_read_only(read_only, secret)
     return 'OK'
   end
 
+  # Checks if the Cassandra seed node is up.
+  def seed_up(secret)
```

Comment:
Should it have a more descriptive name? Like is_db_master_up? or is_cassandra_seed_up?